internal/host/host_test.go

```diff
@@ -3307,3 +3307,163 @@ var _ = Describe("Installation stages", func() {
 		})
 	})
 })
+
+var _ = Describe("sortHost by hardware", func() {
+	var (
+		ctrl    *gomock.Controller
+		diskID1 = "/dev/disk/by-id/test-disk-1"
+		diskID2 = "/dev/disk/by-id/test-disk-2"
+		diskID3 = "/dev/disk/by-id/test-disk-3"
+	)
+	BeforeEach(func() {
+		ctrl = gomock.NewController(GinkgoT())
+	})
+
+	AfterEach(func() {
+		ctrl.Finish()
+	})
+
+	hwSpecifications := []struct {
+		description string
+		Cpus        int64
+		Ram         int64
+		Disks       []*models.Disk
+	}{
+		{
+			description: "minimal master with 3 disks of 120 GB",
+			Cpus:        4,
+			Ram:         16,
+			Disks: []*models.Disk{
+				{SizeBytes: 40 * conversions.GB, DriveType: "HDD", ID: diskID1},
+				{SizeBytes: 40 * conversions.GB, DriveType: "SSD", ID: diskID2},
+				{SizeBytes: 40 * conversions.GB, DriveType: "SSD", ID: diskID3},
```

Comment:
The descriptions don't match the disk sizes (in all cases) - e.g., here it says 3*120GB but the host has 3*40GB

Reply:
I meant accumulative capacity. What would be a better wording?

Reply:
I see.  I assumed it was a mistake because it's less than 100GB.  I guess that ties into the eligible disk comment.